bamboo/select-stack.js

```diff
@@ -31,8 +31,11 @@ function determineIntegrationTestStackName(cb) {
     'Nate Pauzenga': 'np-ci',
     'Naga Nages': 'nnaga-ci',
     vpnguye2: 'vkn-ci',
+  'Roger Kwarteng':'rkwarten-ci',
```

Comment:
Looks like we have some spacing issues. You can actually use our linter to surface things like this:

`npm run lint`

The CI run will fail on this but you can run it locally to get a sneak preview 😄